### PR TITLE
fix: Added Aetheria to the valid world ranges, and synced that value …

### LINKED ISSUE #216: You still cannot use URL position or /goto to go to Aetheria north of Y=150 or east of X=150
None of these work:
In each case you are not taken to 63,151 in Aetheria as expected.

Case 1) 
https://play.decentraland.org?positiion=63,151
//expected:  arrive at 63,151
//actual: you arrive at 0,0

Case 2) you don't move, you get a notice that the world is still limited to 150,150
/goto 63,151
//expected:  arrive at 63,151
//actual: you get a warning that you cannot go outside of 150,150.

Case 3)   
https://play.decentraland.zone/branch/feat/fix-goto-for-non-main-regions/index.html?ENV=org&position=63%2C151&realm=baldr-amber
// expected: arrive 63,151
// actual: you go to 63,150

This includes the fact that decentraland/explorer#1624 is still not fixed in production, but this issue reports error cases that are not all covered in that issue.

- [x] Include parcels outside of the 150, -150 boundaries to include Aetheria parcels
- [x] Fix preview popup in the minimap

### PR DESCRIPTION
**Pull request by kuruk-mm**
…between kernel and renderer

Fix decentraland/unity-renderer#216

# What?
Implement array of world ranges for irregular shapes of the world, and synced that data between kernel and renderer

# Why?
The world is not a regular shape from -150,-150 to 150,150, there are some corners to be aware.

### CODE REVIEW

**Comment by kuruk-mm** [comment]
This PR is blocked by decentraland/common-schemas#6

File: unity-client/Assets/Scripts/MainScripts/DCL/Controllers/MapRenderer/MapRenderer.cs [hunk 1]
```diff
@@ -252,9 +261,22 @@ void UpdateParcelHold()
             }
         }
 
+        private void OnKernelConfigChanged(KernelConfigModel current, KernelConfigModel previous)
+        {
+            Debug.Log("OnKernelConfigChanged");
```
Comment (BrianAmadori) [inline unity-client/Assets/Scripts/MainScripts/DCL/Controllers/MapRenderer/MapRenderer.cs]:
```suggestion
           
```

File: unity-client/Assets/Scripts/MainScripts/DCL/Controllers/MapRenderer/MapRenderer.cs [hunk 2]
```diff
@@ -252,9 +261,22 @@ void UpdateParcelHold()
...
+
+            validWorldRanges = current.validWorldRanges;;
```
Comment (BrianAmadori) [inline unity-client/Assets/Scripts/MainScripts/DCL/Controllers/MapRenderer/MapRenderer.cs]:
```suggestion
            validWorldRanges = current.validWorldRanges;
```
Remove the extra semi-colon.

File: unity-client/Assets/Scripts/MainScripts/DCL/Controllers/MapRenderer/MapRenderer.cs [hunk 3]
```diff
@@ -252,9 +261,22 @@ void UpdateParcelHold()
...
+        }
+
         bool CoordinatesAreInsideTheWorld(int xCoord, int yCoord)
         {
-            return (Mathf.Abs(xCoord) <= WORLDMAP_WIDTH_IN_PARCELS / 2) && (Mathf.Abs(yCoord) <= WORLDMAP_WIDTH_IN_PARCELS / 2);
+            foreach(WorldRange worldRange in validWorldRanges)
+            {
+                if (worldRange.Contains(xCoord, yCoord)) {
```
Comment (BrianAmadori) [inline unity-client/Assets/Scripts/MainScripts/DCL/Controllers/MapRenderer/MapRenderer.cs]:
```suggestion
                if (worldRange.Contains(xCoord, yCoord)) {
```
This file needs formatting to bring the braces after the new line.

**Review by BrianAmadori — APPROVED** [review]
LGTM codewise. Just left some minor remarks that'd be nice if you can resolve it before the merge.

**Pull request merged**